accruals_settings.xml

```diff
@@ -1,28 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <settings xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.1.0 http://maven.apache.org/xsd/settings-1.1.0.xsd" xmlns="http://maven.apache.org/SETTINGS/1.1.0"
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
     <servers>
         <server>
             <id>callisto-jparest</id>
-            <configuration>
-                <httpHeaders>
-                    <property>
-                        <name>Authorization</name>
-                        <value>Bearer ${ARTIFACTORY_TOKEN}</value>
-                    </property>
-                </httpHeaders>
-            </configuration>
+            <!-- Any username would work as long as it's not empty -->
+            <username>DummyUser</username>
+            <password>${env.GITHUB_TOKEN}</password>
         </server>
     </servers>
 
     <profiles>
         <profile>
-            <id>artifactory</id>
+            <id>github</id>
             <repositories>
                 <repository>
                     <id>callisto-jparest</id>
-                    <name>Callisto Jparest Repo</name>
-                    <url>https://artifactory.digital.homeoffice.gov.uk/artifactory/callisto-jparest</url>
+                    <name>Callisto kafka commons Repo</name>
```

Comment:
This name `Callisto kafka commons` doesn't tie up with the github package repository `callisto-jparest`

Reply:
Good spot - Fixed